### ISSUE TITLE
Error: lexical error: invalid char in json text for invalid locations

### ISSUE THREAD

**Issue by SridharJagannathan** [jefferis/fafbseg#12]
```
library(fafbseg)
p.flywire.raw <- cbind(118865, 71338, -2267)
p.flywire.nm <- p.flywire.raw * c(4,4,40)
fafbseg:::mapmany(p.flywire.nm)

```
produces, 

```
Error: lexical error: invalid char in json text.
                                [{"dx":NaN,"dy":NaN,"x":NaN,"y":NaN,"z
                     (right here) ------^ 
```
The `mapmany` function should handle NaN and produce warnings for invalid locations.

The usual usage from the end-user perspective is:

```
flywire2fafb(p.flywire.nm, method = "mapmany")
```
produces,

```
      X  Y      Z
[1,] NA NA -90680
Warning message:
In mapmany(xyzraw, ...) : Internal Server Error (HTTP 500).

```

**Comment by SridharJagannathan** [comment]
@jefferis can you please check and merge this PR?

**Comment by jefferis** [comment]
Closed by #13. Thank you @SridharJagannathan!
